### ISSUE TITLE
Can i use TransR ?

### ISSUE THREAD

**Issue by jaiabhayk** [facebookresearch/PyTorch-BigGraph#218]
Hi,

I am curious if i can use TransR (if yes, please can you provide some pointers or sample code for that ?)?

Thanks

**Comment by lw** [comment]
I'm not familiar with TransR. The PBG approach is to provide modular components that can be put together in various combinations to form full models (these are operators, comparators, loss functions, ...; you can see them all in our docs). If TransR "fits" this decomposition schema used by PBG then you should be able to recreate it. If you find that some component is missing it shouldn't be too to write it in which case please consider contributing it upstream!

**Comment by jaiabhayk** [comment]
I think, it seems TransR would not be that computationally efficient (primary reason PBG is built for), but will let you know/contribute if that makes sense to add in PBG. 

Thanks!
